content/quickstart/python/metrics.md

```diff
@@ -268,12 +271,338 @@ def readEvaluateProcessLine():
     # Insert the tag map finally
     mmap.record(tmap)
 
+if __name__ == "__main__":
+    main()
+
+{{</highlight>}}
+
+### Register Views
+We will create a function called `setupOpenCensusAndPrometheusExporter` and call it from our main function:
+
+{{<tabs Snippet All>}}
+{{<highlight python>}}
+def main():
+    setupOpenCensusAndPrometheusExporter()
+    while True:
+        readEvaluateProcessLine()
+
+def registerAllViews(view_manager):
+    view_manager.register_view(latency_view)
+    view_manager.register_view(line_count_view)
+    view_manager.register_view(error_count_view)
+    view_manager.register_view(line_length_view)
+
+def setupOpenCensusAndPrometheusExporter():
+    stats = stats_module.Stats()
+    view_manager = stats.view_manager
+
+    registerAllViews(view_manager)
+{{</highlight>}}
+
+{{<highlight python>}}
+#!/usr/bin/env python
+
+import sys
+import time
+
+from opencensus.stats import stats
+from opencensus.stats.exporters import prometheus_exporter as prometheus
+
+from opencensus.stats import aggregation as aggregation_module
+from opencensus.stats import measure as measure_module
+from opencensus.stats import view as view_module
+from opencensus.stats import stats as stats_module
+from opencensus.tags import tag_key as tag_key_module
+from opencensus.tags import tag_map as tag_map_module
+from opencensus.tags import tag_value as tag_value_module
+
+
+# Create the measures
+# The latency in milliseconds
+m_latency_ms = measure_module.MeasureFloat("repl/latency", "The latency in milliseconds per REPL loop", "ms")
+
+# Counts the number of lines read in from standard input
+m_lines_in = measure_module.MeasureInt("repl/lines_in", "The number of lines read in", "1")
```

Comment:
Not needed, we can get line count by apply the Count Aggregation on `m_latency_ms` or on `m_line_lengths`. Please visit the updated stats guides for other languages.